src/styles/transitions/_text-page-load.scss

```diff
@@ -0,0 +1,22 @@
+.text-page-load {
+  max-width: 590px;
+  display: block;
+  margin: auto;
+}
+
+.text-page__title-load{
+  height: 32px;
+  margin: 0 0 0.5rem 0;
+  border-radius: 10px;
+  background-color: $light-grey;
```

Comment:
The color seems pretty dark to me. You can create a new SCSS variable in `styles/abstracts/_variables.scss` called `$transition-grey` or something like that and have it be lighter to resemble the image in the issue (linked above)